k8s-ruby.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.bindir        = "bin"
   spec.executables   = []
   spec.require_paths = ["lib"]
-  spec.required_ruby_version = [">= 2.4", "< 3.4"]
+  spec.required_ruby_version = [">= 2.4", "< 3.5"]
```

Comment:
Maybe we could just remove the upper constraint here altogether so this doesn't happen every year? 

https://github.com/grpc/grpc/blob/master/grpc.gemspec#L16